misk-clustering-dynamodb/src/main/kotlin/misk/clustering/dynamo/DynamoCluster.kt

```diff
@@ -1,36 +0,0 @@
-package misk.clustering.dynamo
-
-import misk.clustering.Cluster
-import misk.clustering.Cluster.Member
-import misk.clustering.Cluster.Snapshot
-import misk.clustering.ClusterHashRing
-import misk.clustering.ClusterWatch
-
-class DynamoCluster(self: String) : Cluster {
```

Comment:
Do we have an idea of the scope of what downstream repos might not be compatible with this class being removed?

Reply:
This was a recently added module and it's not being used yet.